ttmar_starter.html

```diff
@@ -54,12 +54,41 @@
   <p class="second_chorus" style="display:none">Go on take the money and run</p>
   <p class="second_chorus" style="display:none">Go on take the money and run</p>
   <p class="second_chorus" style="display:none">Go on take the money and run</p>
-  <p class="second_chorus" style="display:none">Go on take the money and run</p>
+  <p class="second_chorus" style="display:none">Go on take the money and run
```

Comment:
Be careful about changing the markup if you didn't mean to! This could cause problems!